cmd/env.go

```diff
@@ -95,7 +95,17 @@ func init() {
 	//revive:disable:line-length-limit
 	envCmd.Flags().StringP("account", "a", "", "Account ID to assume role on (defaults to source account)")
 	envCmd.Flags().StringP("session", "s", "", "Set session name for assumed role (defaults to origin user name)")
-	envCmd.Flags().Int64P("lifetime", "l", creds.SessionLifetimeDefault, fmt.Sprintf("Set lifetime of credentials in seconds between %d and %d", creds.SessionLifetimeMin, creds.SessionLifetimeMax))
+	envCmd.Flags().Int64P(
+		"lifetime", "l",
+		0,
```

Comment:
It seems fine to remove this default as there is special logic to detect when the options struct has a zero value. But, perhaps in that case it would be good to revert to specifying the default explicitly in the help text (I believe only non-zero values are automatically documented by cobra). Though the help text is a good deal longer now.

This change also isn't reflected in the README yet 🤷‍♂

Reply:
Yea. I don’t super mind the long help text, but will add the default and update the readme.